<launch>

  <node pkg="lidar_perception" type="lidar_perception_node" name="lidar_perception_node" output="screen">
    <!-- topic input -->
    <param name="in_point_cloud_topic" type="str" value="/driver/livox/point_cloud" />
    <param name="in_localization_topic" type="str" value="/localization/estimation" />
    <!-- topic output -->
    <param name="out_local_grid_map_topic" type="str" value="/perception/local_grid_map" />
    <param name="out_global_objects_topic" type="str" value="/perception/global_objects" />
    <!-- debug parameters, set to true only when debugging -->
    <param name="publish_local_points" type="bool" value="false" />
    <param name="out_local_points_topic" type="str" value="/perception/local_points" />
    <param name="publish_global_points" type="bool" value="false" />
    <param name="out_global_points_topic" type="str" value="/perception/global_points" />
    <!-- parameters from parameters.yaml file -->
    <rosparam file="$(find lidar_perception)/config/perception_params.yaml" command="load" />

  </node>

</launch>
